tests/CMakeLists.txt

```diff
@@ -4,6 +4,20 @@ TARGET_INCLUDE_DIRECTORIES(unittest_nnstreamer-edge PRIVATE ${EDGE_REQUIRE_PKGS_
 TARGET_LINK_LIBRARIES(unittest_nnstreamer-edge ${TEST_REQUIRE_PKGS_LDFLAGS} ${NNS_EDGE_LIB_NAME})
 INSTALL (TARGETS unittest_nnstreamer-edge DESTINATION ${BIN_INSTALL_DIR})
 
+# Custom connection lib
+SET(NNS_EDGE_CUSTOM_LIB_NAME nnstreamer-edge-custom)
```

Comment:
For test?
then, how about changing lib name and file name to nnstreamer-edge-custom-test?

Reply:
Good, Updated!